gui/src/ScenarioSelector.tsx

```diff
@@ -344,30 +498,88 @@ const ScenarioEditDialog = ({ scenario, submit, open, actionEdit, configs, close
               </Grid>
               {properties
                 ? properties.map((item, index) => (
-                    <Grid item xs={12} key={item.id} container justifyContent="space-between">
+                    <Grid
+                      item
+                      xs={12}
+                      key={item.id}
+                      container
+                      spacing={1}
+                      alignItems="center"
+                    >
                       <Grid item xs={4}>
-                        <TextField value={item.key} label="Key" variant="outlined" data-name="key" data-idx={index} onChange={updatePropertyField} />
+                        <TextField
+                          value={item.key}
+                          label="Key"
+                          variant="outlined"
+                          data-name="key"
+                          data-idx={index}
+                          onChange={updatePropertyField}
+                        />
                       </Grid>
-                      <Grid item xs={5}>
-                        <TextField value={item.value} label="Value" variant="outlined" data-name="value" data-idx={index} onChange={updatePropertyField} />
+                      <Grid item xs>
+                        <TextField
+                          value={item.value}
+                          label="Value"
+                          variant="outlined"
+                          data-name="value"
+                          data-idx={index}
+                          onChange={updatePropertyField}
+                        />
                       </Grid>
-                      <Grid item xs={2}>
-                        <Button variant="outlined" component="label" data-id={item.id} onClick={propertyDelete}>
+                      <Grid
+                        item
+                        xs="auto"
+                        sx={{ display: "flex", alignItems: "flex-end" }}
```

Comment:
sx as const

Reply:
Unnecessary after my intervention. Removed it.